lib/rack/response.rb

```diff
@@ -128,6 +128,7 @@ def not_found?;           status == 404;                        end
       def method_not_allowed?;  status == 405;                        end
       def precondition_failed?; status == 412;                        end
       def unprocessable?;       status == 422;                        end
+      def unavailable?;         status == 451;                        end
```

Comment:
This could be confusing with `503 Service Unavailable`. I'd be open to renaming this method to `unavailable_for_legal_reasons`, but that's awfully wordy. Figured I'd start simple.